### DISCUSSION TITLE
UTM parameters

### DISCUSSION THREAD

**Discussion by altingrc** [umami-software/umami#2517] [Q&A]
Somebody find a way to track where links came from? i want to track that this way : 

https://www.mywebsite.com?utm_source=test+source&utm_medium=test+medium&utm_campaign=test+campaign

I didn't see anything in the query parameters tab

**Comment by TreyWW** [comment]
Hey,

Sorry about the late reply. You can view UTM tracking in the UTM report tab https://umami.is/docs/reports/report-utm

Here's all the ways you can track campaigns: [docs](https://umami.is/docs/guides/measure-campaigns)

Hope it helps!

**Reply by altingrc** [reply]
Thanks 🙏🏼
